CHANGELOG.rst

```diff
@@ -5,6 +5,45 @@ Kubernetes Collection Release Notes
 .. contents:: Topics
 
 
+v3.0.0
+======
+
+Release Summary
+---------------
+
+This major release drops support for ansible-core versions lower than 2.14, Python versions lower than 3.9 and updates python kubernetes library to 24.2.0 , helm/kind-action to 1.8.0, kubernetes >= 1.24, along with bug fixes and minor changes.
+
+
+Minor Changes
+-------------
+
+- helm - add reuse_values and reset_values support to helm module (https://github.com/ansible-collections/kubernetes.core/issues/394).
+- k8s - add new option delete_all to support deletion of all resources when state is set to absent. (https://github.com/ansible-collections/kubernetes.core/issues/504)
+- k8s, k8s_info - add a hidden_fields option to allow fields to be hidden in the results of k8s and k8s_info
+- k8s_drain - add ability to filter the list of pods to be drained by a pod label selector (https://github.com/ansible-collections/kubernetes.core/issues/474).
+
+Breaking Changes / Porting Guide
```

Comment:
Should we move the Breaking Changes section to the top (in config.yaml)?

Reply:
Is this ordering done by antsibull-changelog tool? I see this is the order maintained in the previous releases as well.

Reply:
Yeah it is but the order is set by the order of sections in the changelog/config.yaml file.

Reply:
I don't know why I thought I saw that at the top in other collections but now I'm not seeing so I don't know where that came from :)